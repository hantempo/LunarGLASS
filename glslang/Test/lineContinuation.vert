#version 300 es

// this file cont\
ains no errors other than the #error which are there to see if line numbering for errors is correct

#error e1

float f\
oo;  // same as 'float foo;'

#error e2

#define MAIN void main() \
   {                     \
gl_Position = vec4(foo); \
} 

#error e3

MAIN

vec4 foo2(vec4 a)
{                                
  vec4 b = a;       \
  return b;                   
}

// aoeuntheo unatehutna \ antaehnathe 
// anteonuth $ natohe " '
// anteonuth     natohe
