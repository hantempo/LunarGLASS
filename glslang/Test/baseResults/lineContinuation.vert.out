lineContinuation.vert
ERROR: 0:6: '#error' : e1  
ERROR: 0:11: '#error' : e2  
ERROR: 0:18: '#error' : e3  
ERROR: 3 compilation errors.  No code generated.


ERROR: node is still EOpNull!
0:20  Function Definition: main( (void)
0:20    Function Parameters: 
0:20    Sequence
0:20      move second child to first child (highp 4-component vector of float)
0:20        'gl_Position' (gl_Position highp 4-component vector of float)
0:20        Construct vec4 (highp 4-component vector of float)
0:20          'foo' (highp float)
0:22  Function Definition: foo2(vf4; (highp 4-component vector of float)
0:22    Function Parameters: 
0:22      'a' (in highp 4-component vector of float)
0:24    Sequence
0:24      Sequence
0:24        move second child to first child (highp 4-component vector of float)
0:24          'b' (highp 4-component vector of float)
0:24          'a' (in highp 4-component vector of float)
0:25      Branch: Return with expression
0:25        'b' (highp 4-component vector of float)
0:?   Linker Objects
0:?     'foo' (highp float)
0:?     'gl_VertexID' (gl_VertexId highp int)
0:?     'gl_InstanceID' (gl_InstanceId highp int)


Linked vertex stage:



